### ISSUE TITLE
Queued integrations can hold onto invalid token IDs

### ISSUE THREAD

**Issue by Mosnar** [verbb/formie#1861]
### Describe the bug

I've found some Salesforce submissions in the queue that are failing to push because the token is no longer valid - it's null. Even if I procure a new token, the failed jobs hold onto the null token and fail to resubmit.

```
API error: “Call to undefined method TypeError::getResponse()” vendor/verbb/formie/src/integrations/crm/Salesforce.php:409
```

### Steps to reproduce

1. Create an integration with a token (such as Salesforce)
2. Authenticate it and push some submissions
3. Invalidate/remove the tokens before the queue finishes
4. See failure

### Form settings

N/A

### Craft CMS version

Craft CMS 3.8.5

### Plugin version

1.6.26

### Multi-site?

Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-3 as 1.6.41"`.

**Comment by engram-design** [comment]
Fixed in [1.6.42](https://github.com/verbb/formie/releases/tag/1.6.42)
